src/Raven.Server/Utils/Cli/BrowserHelper.cs

```diff
@@ -6,22 +6,38 @@ namespace Raven.Server.Utils.Cli
 {
     public static class BrowserHelper
     {
-        public static void OpenStudioInBrowser(string url)
+        public static bool OpenStudioInBrowser(string url, Action<object> onError = null)
         {
             try
             {
                 if (PlatformDetails.RunningOnPosix == false)
                 {
                     Process.Start(new ProcessStartInfo("cmd", $"/c start {url}"));
                 }
+                else if (PlatformDetails.RunningOnMacOsx)
```

Comment:
In 4.1 there is a similar method in EmbeddedServer. Can you apply this change there also?